### DISCUSSION TITLE
CoolWallet Coin Tester ERROR

### DISCUSSION THREAD

**Discussion by fantersee** [CoolBitX-Technology/coolwallet-signing-tools#45] [Q&A]
- Description

   執行 coin-template 的 sign transaction 出現錯誤訊息，我的卡版本320

- Screenshots or appendi
![S__276078596](https://user-images.githubusercontent.com/4178903/151511943-bc36afc9-7532-4a67-adf5-e7cf39180ca0.jpg)
x

**Comment by hankliu418** [comment]
script的header格式不對，第一個byte表示header的長度，只會有03跟04這兩種情況
